userpanel/modules/documents/style/bclean/templates/documents.html

```diff
@@ -18,8 +18,19 @@
 		margin-bottom: 0.6em;
 	}
 
+	tr.table-click:hover {
+		cursor: pointer;
+	}
+
+	h5 {
+		margin-top: 10px;
+		margin-bottom: 10px;
+		font-weight: bold;
+	}
 </style>
 
+<h5>{trans("Documents:")}</h5>
```

Comment:
Tp juz kilka razy było przemycane i inni prosili o wywalenie tego dodatkowego, zbytecznego wiersza.

Reply:
Wszędzie tak mamy, że mamy nagłówek nad tabelą.
Chcesz, żebym wyrzucił ze wszystkich modułów taką linię czy tylko tutaj?

Reply:
W***rz to wszędzie i jakoś wyróżnić innym tłem w pasku zakładek aktualnie wybraną zakładkę. Będzie dużo lepiej. Podejrzewam, że podobnie będzie uważał @awbnet (choć mogę się mylić :)).

Reply:
Tak patrzę na screeny i przychylam się do @chilek. Na górnej belce jest podświetlone gdzie akurat jesteśmy, więc po co dublować? 🤔

Reply:
Mi też się tak kompaktowo bardziej podoba. Zrobione.

Reply:
@chilek to co puścisz to?